setup.ts

```diff
@@ -95,38 +177,63 @@ async function askForRedisUrl(): Promise<{
 }
 
 // get the redis url
+/**
+ * The `redisConfiguration` function updates the Redis configuration by prompting the user for the
+ * Redis URL, checking the connection, and updating the environment variables and .env file
+ * accordingly.
+ */
 async function redisConfiguration(): Promise<void> {
   const REDIS_URL = process.env.REDIS_URL;
 
   try {
     let isConnected = false,
       url = "";
+    let host!: string;
+    let port!: number;
+    let password!: string;
+
     while (!isConnected) {
-      const { host, port, password } = await askForRedisUrl();
+      const result = await askForRedisUrl();
+      host = result.host;
+      port = result.port;
+      password = result.password;
+
       url = `redis://${password ? password + "@" : ""}${host}:${port}`;
       isConnected = await checkRedisConnection(url);
     }
 
+    // Set the Redis parameters in process.env
+    process.env.REDIS_URL = url;
+    process.env.REDIS_HOST = host;
+    process.env.REDIS_PORT = port.toString();
+    process.env.REDIS_PASSWORD = password;
+
+    // Update the .env file
     const config = dotenv.parse(fs.readFileSync(".env"));
     config.REDIS_URL = url;
-    fs.writeFileSync(".env", "");
-    for (const key in config) {
-      fs.appendFileSync(".env", `${key}=${config[key]}\n`);
-    }
+    config.REDIS_HOST = host;
+    config.REDIS_PORT = port;
+    config.REDIS_PASSWORD = password;
+    updateEnvVariable(config);
+    console.log("\nRedis configuration updated successfully!");
   } catch (err) {
     console.error(err);
     abort();
   }
 }
 
 //LAST_RESORT_SUPERADMIN_EMAIL prompt
+/**
+ * The function `askForSuperAdminEmail` asks the user to enter an email address and returns it as a promise.
+ * @returns The email entered by the user is being returned.
+ */
 async function askForSuperAdminEmail(): Promise<string> {
   const { email } = await inquirer.prompt([
     {
       type: "input",
       name: "email",
       message:
-        "Enter the email which you wish to assign as the Super Admin of last resort:",
+        "Please make sure to register with this email before logging in.\n Enter the email which you wish to assign as the Super Admin of last resort :",
```

Comment:
This has the format of all the other questions. 

1. It is a statement, so it shouldn't start with a `?`
2. The bold text also makes it difficult to differentiate between other questions. All other non-questions are not bold
3. There should be spaces above and below the text to highlight this fact

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/543aa69e-564c-49f4-841a-221c97a31a3f)